cmd/tapcli/assets.go

```diff
@@ -425,9 +473,37 @@ func sealBatch(ctx *cli.Context) error {
 	client, cleanUp := getMintClient(ctx)
 	defer cleanUp()
 
-	resp, err := client.SealBatch(ctxc, &mintrpc.SealBatchRequest{
+	req := &mintrpc.SealBatchRequest{
 		ShortResponse: ctx.Bool(shortResponseName),
-	})
+	}
+
+	// TODO(guggero): Actually just ask for the signed PSBT back and extract
```

Comment:
Will this be addressed in this PR? 

So accepting the signed PSBT to extract the signatures manually ourselves.

Reply:
One other related thing is that we want to add some basic meta data to the group key minting PSBT to allow external software to display information such as the asset ID, name, and units to be minted.

Reply:
> Will this be addressed in this PR?
> 
> So accepting the signed PSBT to extract the signatures manually ourselves.

It will be addressed here: https://github.com/lightninglabs/taproot-assets/pull/1295

Reply:
> One other related thing is that we want to add some basic meta data to the group key minting PSBT to allow external software to display information such as the asset ID, name, and units to be minted.

I've put up this issue: https://github.com/lightninglabs/taproot-assets/issues/1298